docs/documentation/maya/skin.md

```diff
@@ -1,127 +1,183 @@
+    - Has a range of \[0.0, 2.0\] [^1]
+ - **Stretching Multiplier** (Float, 1.0): Sets the scaling factor applied to the stretching resistance of every point.
+    - Has a range of \[0.0, 2.0\] [^1]
+ - **Attenuation Velocity Factor** (Float, 1.0): Sets the weight of the attenuation applied to the whole simulation driven by the Attenuation Matrix.
+    - Has a range of \[0.0, 10.0\] [^1]
+ - **Space Scale Mode** (Enumerator, "Masses + Forces"): Defines the mode of execution for the sliding constraints.
+    - *Quality* is more accurate, recommended for final results.
+    - *Fast* provides higher performance, recommended for preview.
+
+<!-- Removed additional attributes as they were basically paintable weights, previously explained  -->
 
 ## Attribute Editor Template
 
 <figure markdown>
   ![skin editor first part](images/attribute_editor_part_one_skin.png) 
-  <figcaption>Figure 1: Skin Attribute Editor</figcaption>
+  <figcaption>Figure 1: AdnSkin Attribute Editor</figcaption>
 </figure>
 
 <figure markdown>
   ![skin editor second part](images/attribute_editor_part_two_skin.png)
-  <figcaption>Figure 2: Skin Attribute Editor (Advanced Settings)</figcaption>
+  <figcaption>Figure 3: AdnSkin Attribute Editor (Advanced Settings)</figcaption>
+</figure>
+
+## Debugger
+
+To better visualize deformer constraints and attributes in the Maya viewport there is the option to enable the debugger, found in the dropdown menu labeled "Debug" in the attribute editor.
+
+To enable the debugger the *Debug* checkbox must be marked. To select the specific feature you would like to visualize, choose it from the list provided in *Features*. 
+
+### Debug features
+
+The features that can be visualized with the debugger in the AdnSkin deformer are:
+
+ - **Hard Constraints**: For each vertex, a line will be drawn from the simulated mesh to its corresponding reference point on those vertices where its Hard Constraints weight is greater than 0.0.
+ - **Soft Constraints**: For each vertex, a line will be drawn from the simulated mesh to its corresponding reference point on those vertices where its Soft Constraints weight is greater than 0.0.
+ - **Slide Constraints**: For each vertex, a line will be drawn from the simulated mesh to its corresponding reference point on those vertices where its Soft Constraints weight is greater than 0.0.
+
+Enabling the debugger and selecting one of these constraints will draw lines from the influenced vertices in the simulated mesh to their corresponding reference vertices. 
+
+<figure markdown>
+  ![skin editor debug menu](images/skin_debug.png)
+  <figcaption>Figure 5: Debugger enabled displaying hard constraints, slide constraints and soft constraints with different configurations. </figcaption>
+</figure>
+
+### Debugger attributes
+
+The following attributes can be modified to better customize the appereance of these lines:
+
+ - **Width Scale** (Float, 3.0): Modifies the width of all lines.
```

Comment:
Please also add `Debug` and `Feature` to the list of attributes

Reply:
Could you take care of this specific change? I don't know exactly how to define those attributes.

Reply:
Let me write it here (It's hard to format here so please make sure that everything is tabulated correctly:
```suggestion
 - **Debug** (Boolean, False): Enable or Disable the debug functionalities in the viewport for the AdnSkin deformer.
 - **Feature** (Enum, "Hard Constraints"): A list of debuggable features for this deformer.
     - Hard Constraints: Draw hard constraints connections from the simulated mesh to the reference mesh.
     - Soft Constraints: Draw soft constraints connections from the simulated mesh to the reference mesh.
     - Slide Constraints: Draw slide constraints connections from the simulated mesh to the reference mesh.
 - **Width Scale** (Float, 3.0): Modifies the width of all lines.
```